<?xml version="1.0"?>

<launch>

<!--  rslidar pointcloud parse node -->
<node  name="cloud_node" pkg="rslidar_pointcloud" type="cloud_node" output="screen" >
	<param name="model" value="RS32"/>
	<!--param name="frame_id" value="rs_odom" /-->
	<param name="hori_angel_begin" value="0"/>
	<param name="hori_angel_end" value="0"/>   
	<param name="curves_path" value="$(find rslidar_pointcloud)/data/rs_lidar_32/B19/curves.csv" />
	<param name="angle_path" value="$(find rslidar_pointcloud)/data/rs_lidar_32/B19/angle.csv" />
	<param name="channel_path" value="$(find rslidar_pointcloud)/data/rs_lidar_32/B19/ChannelNum.csv" />
</node>

<!--main testing node-->
  <node pkg="robosense_sdk" type="test_without_map"  name="test_without_map"  output="screen" >
    <param name="key_path" value="$(find robosense_sdk)/key" />
    <param name="pcap" value="$(find robosense_sdk)/data/validation.pcap" />
    <param name="ethernet_name" value="eth0" />	
    <param name="port" value="6699" /> <!--for online validation only-->
    <param name="args_path" value="$(find robosense_sdk)/usr_args/perception_args_32_withoutmap.xml" />
	<param name="lidar_config_path" value="$(find auto_align_tool)/save_config/align.txt" />
  </node>

<!--car module for ui display-->
  <group>
    <arg name="gui" default="False" />
    <param name="use_gui" value="$(arg gui)"/>
    <param name="robot_description" textfile="$(find robosense_sdk)/car_rviz_model/no_map/default.urdf" />
    <node pkg="joint_state_publisher" type="joint_state_publisher" name="joint_state_publisher"  />
    <node pkg="robot_state_publisher" type="state_publisher" name="robot_state_publisher"  />
  </group>

<!--rviz show-->
  <node pkg="rviz" type="rviz" name="rviz"  args="-d $(find robosense_sdk)/cfg/robosense_no_map_test.rviz"/>


</launch>
